authority/voting/server/state_test.go

```diff
@@ -574,11 +587,15 @@ func genMixConfig(name string, pki *sConfig.PKI, port uint16) (*identityKey, *sC
 }
 
 // create epoch keys
-func genMixKeys(votingEpoch uint64) map[uint64]*ecdh.PublicKey {
-	mixKeys := make(map[uint64]*ecdh.PublicKey)
+func genMixKeys(votingEpoch uint64) map[uint64][]byte {
+	mixKeys := make(map[uint64][]byte)
 	for i := votingEpoch; i < votingEpoch+2; i++ {
-		idKey, _ := ecdh.NewKeypair(rand.Reader)
-		mixKeys[i] = idKey.PublicKey()
+		_, pubkey, err := ecdh.EcdhScheme.GenerateKeyPairFromEntropy(rand.Reader)
```

Comment:
MixKeys must be bytes so that it's generic. For this specific test it doesn't matter the contents of MixKeys, they don't need to be keys but they are signed in the test.